src/ios/Accessibility.ts

```diff
@@ -89,6 +89,11 @@ export class Accessibility extends NativeAccessibility {
             }
         }
     }
+
+    private _compareRawAnnouncements(postedAnnouncement: string, payloadAnnouncement: string): boolean {
+        const spacesCommasRegExp = /[s,]/g; // special regex for iOS 12
```

Comment:
This regex matches the 's' character and ',' comma

>test, foo

will get transformed to

> tet foo

Meaning tet, foo will incorrectly "match" test, foo

Reply:
@erictraut - this PR requires changes before you publish next reactxp

Reply:
@berickson1 my bad - mistake during CR fixes - will create a new PR